### ISSUE TITLE
ListIncidents() essentially ignores ListIncidentsOptions.Includes parameter

### ISSUE THREAD

**Issue by rtfb** [PagerDuty/go-pagerduty#128]
When I call `ListIncidents` with an option to include the trigger log entry:

```go
	o := pagerduty.ListIncidentsOptions{
		// ...
		Includes: []string{"first_trigger_log_entries"},
	}
```

I get back `Incident` structures, which have `FirstTriggerLogEntry` as `APIObject`, which is only a reference, not a joined object. As far as I can tell looking at the code and experimenting with `curl` calls, the actual response does include log entries as expected, they just get dropped by `json.Decoder.Decode`, because there's nothing for them to deserialize into.

I don't see how this could be addressed by the library user. Can this get fixed internally? Perhaps by introducing an `interface{}` field to `APIObject`?

**Comment by abtris** [comment]
👍 I have the same problem, I need get assignee and getting empty array.

**Comment by abtris** [comment]
I think maybe https://github.com/PagerDuty/go-pagerduty/pull/93 can solve problem.

**Comment by abtris** [comment]
Still not resolved getting Assignments and Acknowledgements as empty arrays.

**Comment by theckman** [comment]
I think the issue @rtfb reported was fixed via 4f6a061e0d3750cb80a7629f4cd0673360976b8c and b6a682c4a68cbdc6c69f3f23757f6c07fa30718e. So I'm going to close this issue.

**Comment by theckman** [comment]
@abtris I'm not sure I fully grok the challenge you're experiencing. Would you be able to file a new issue if you're still experiencing it.
